jobs/jenkins_jobs_pr.groovy

```diff
@@ -0,0 +1,65 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = 'jenkins-jobs-pr'
+
+job(name) {
+  description """
+    <ol>
+      <li>Watches the <a href="https://github.com/deis/jenkins-jobs">jenkins-jobs</a> repo for a PR commit</li>
+      <li>And runs tests against changes</li>
+    </ol>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/jenkins-jobs")
+        credentials('597819a0-b0b9-4974-a79b-3a5c2322606d')
+      }
+      refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+      branch('${sha1}')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    githubPush()
+
+    pullRequest {
+      admin('deis-admin')
+      cron('H/5 * * * *')
+      useGitHubHooks()
+      triggerPhrase('OK to test')
+      orgWhitelist(['deis'])
+      allowMembersOfWhitelistedOrgsAsAdmin()
+      extensions {
+        commitStatus {
+          context('ci/jenkins/pr')
+          triggeredStatus("Triggering ${name} tests...")
+          startedStatus("Starting ${name} tests...")
+          completedStatus('SUCCESS', "Success! All tests passed.")
+          completedStatus('FAILURE', 'Build/deploy returned failure(s).')
+          completedStatus('ERROR', 'Something went wrong.')
+        }
+      }
+    }
+  }
+
+  wrappers {
+    timestamps()
+    colorizeOutput 'xterm'
+  }
+
+  steps {
+    shell '''
+      #!/usr/bin/env bash
+
+      set -eo pipefail
+
+      make docker-test
```

Comment:
@sgoings ah good catch, thank you!  I've updated change to `cd` into the `bash` subdir for now.  Was thinking to keep the `Makefile` there for now as all targets relate to that particular subdir.